internal/app/components/timelines/timelines_test.go

```diff
@@ -456,3 +456,117 @@ func TestTimeline_InvalidCourses(t *testing.T) {
 		})
 	}
 }
+
+func TestTimeline_NewTimeline(t *testing.T) {
```

Comment:
this function name should be `TestNewTimeline` since it's not a method

Reply:
Oh that makes a lot of sense